app/uploaders/activity_insight_file_uploader.rb

```diff
@@ -18,4 +19,16 @@ def cache_dir
       Rails.root.join "uploads/cache/activity_insight_file_uploads/#{model&.id || object_id}/file"
     end
   end
+
+  def model_object_dir
+    if Rails.env.test?
+      Rails.root.join "tmp/uploads/activity_insight_file_uploads/#{model&.id || object_id}"
+    else
+      Rails.root.join "uploads/activity_insight_file_uploads/#{model&.id || object_id}"
+    end
+  end
+
+  def remove_file
```

Comment:
I think we should rename this to something that indicates it's removing the directory.  Technically, it's not removing the file.